variety/plugins/builtin/display_modes/LegacyDisplayModesPlugin.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+
+from variety.plugins.IDisplayModesPlugin import DisplayMode, IDisplayModesPlugin, StaticDisplayMode
+from variety.Util import _
+
+
+class LegacyDisplayModesPlugin(IDisplayModesPlugin):
+    @classmethod
+    def get_info(cls):
+        return {
+            "name": "LegacyDisplayModesPlugin",
+            "description": "Legacy display mode for compatibility with past Variety versions",
+            "version": "1.0",
+            "author": "Peter Levi",
+        }
+
+    def display_modes(self) -> List[DisplayMode]:
+        return [
+            StaticDisplayMode(
+                id="os",
+                title=_("[Legacy] Controlled via OS settings, not by Variety. Fast."),
```

Comment:
I'm not sure we ought to call this legacy - I think having OS mode as a specific option is nice to have (since it has the least overhead), especially if you know that most of your image sources behave in a way that e.g. makes zoom / centre reasonable.